grpc/sa-wrappers.go

```diff
@@ -283,6 +283,26 @@ func (sac StorageAuthorityClientWrapper) PreviousCertificateExists(
 	return exists, err
 }
 
+func (sac StorageAuthorityClientWrapper) AddPrecertificate(
+	ctx context.Context,
+	req *sapb.AddCertificateRequest,
+) (*corepb.Empty, error) {
+	if req == nil {
+		return nil, errIncompleteResponse
+	}
+	return sac.inner.AddPrecertificate(ctx, req)
+}
+
+func (sac StorageAuthorityClientWrapper) AddSerial(
+	ctx context.Context,
+	req *sapb.AddSerialRequest,
+) (*corepb.Empty, error) {
+	if req == nil {
```

Comment:
Same thought here: the `sac.inner.AddSerial` is going to blindly de-ref the `*sapb.AddSerialRequest` pb fields and each should probably be checked as `!= nil` here too.